public_html/script/src/drawable/root-drawable.js

```diff
@@ -0,0 +1,68 @@
+/*
+    A drawable to be used as the root drawable (directly on the canvas)
+*/
+'use strict';
+
+import {Drawable} from './drawable';
+import {setCanvas} from '../canvas';
+import {GridManager} from '../grid+draggable/grid-manager';
```

Comment:
They're together because grid only really exists for the draggables, so logically they can be put in one group IMO; and otherwise we'd have two really small folders. Yeah + is a little weird; I couldn't think of another character to use